.github/workflows/pr-merge.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   ci-job:
     name: Checks
     uses: ./.github/workflows/checks.yml
-    secrets: inherit
+    # secrets: inherit
```

Comment:
not critical, but some files have `exasol/python-toolbox/.github/actions/python-environment@1.0.1`:
- test-integration-no-db.yml
- test-integration-db.yml
- check-code-generation.yml